### PR TITLE
Update help text to include new commands

### PR DESCRIPTION
**Pull request by gavanderhoorn**
This updates the text that is printed when users run the `help` command.

See a3df575 for the descriptions I've added. I'm not a daily user of the commander, so I'd appreciate it if someone could carefully check them :).

07ef954 is optional: it sorts the text alphabetically. I think it makes sense (facilitates looking for a command in the text), but I can remove it from this PR.

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
Ping?

**Comment by gavanderhoorn** [comment]
:bell: ?

**Comment by davetcoleman** [comment]
I don't use python / commander, but if someone who does could +1 this then I'd merge it

File: src/moveit_commander/interpreter.py [hunk 1]
```diff
@@ -557,16 +557,26 @@ def get_help(self):
         res.append("  go <dir> <dx>|      plan and execute a motion in direction up|down|left|right|forward|backward for distance <dx>")
         res.append("  go rand             plan and execute a motion to a random state")
         res.append("  plan <name>         plan a motion to the state <name>")
+        res.append("  plan rand           plan a motion to a random state")
+        res.append("  planner <name>      use planner <name> to plan next motion")
         res.append("  execute             execute a previously computed motion plan")
+        res.append("  stop                stop the active group")
+        res.append("  pick <name>         pick up object <name>")
+        res.append("  place <name>        place object <name>")
+        res.append("  stop                stop any execution, if active")
```
Comment (130s) [inline src/moveit_commander/interpreter.py]: <s>I see two `stop`s (another one at https://github.com/ros-planning/moveit_commander/pull/42/commits/a3df575bee1d1d22c204203d68ae56fc818ddffb#diff-3fad63fac0e8c61de84350f6122f8bceR563)) with different description. Is this correct?</s>

File: src/moveit_commander/interpreter.py [hunk 2]
```diff
@@ -557,16 +557,26 @@ def get_help(self):
         res.append("  go <dir> <dx>|      plan and execute a motion in direction up|down|left|right|forward|backward for distance <dx>")
         res.append("  go rand             plan and execute a motion to a random state")
         res.append("  plan <name>         plan a motion to the state <name>")
+        res.append("  plan rand           plan a motion to a random state")
+        res.append("  planner <name>      use planner <name> to plan next motion")
         res.append("  execute             execute a previously computed motion plan")
+        res.append("  stop                stop the active group")
```
Comment (130s) [inline src/moveit_commander/interpreter.py]: Looking at the source (probably around [here](https://github.com/kunal15595/ros/blob/681a8810e7264d153a9e4950873ec7afe94767f0/moveit/src/moveit_ros/planning_interface/move_group_interface/src/move_group.cpp#L694)?), this line may be incorrect and [this line](https://github.com/ros-planning/moveit_commander/pull/42/commits/a3df575bee1d1d22c204203d68ae56fc818ddffb#diff-3fad63fac0e8c61de84350f6122f8bceR566) is correct (i.e. no stopping feature for active group)?

File: src/moveit_commander/interpreter.py (hunk 1 above)
Comment (130s) [inline src/moveit_commander/interpreter.py]: Sorry, my comment https://github.com/ros-planning/moveit_commander/pull/42/commits/a3df575bee1d1d22c204203d68ae56fc818ddffb#r69403042 is invalid (I realized that I was looking at a single commit page. You already fixed it in a subsequent commit).

**Comment by 130s** [comment]
+1
Added description looks all good to me and I'm a fan of alphabetization. 

---

The following has nothing to do with review.

Sorry for ignorance but can you teach me how to see this portion of the code gets called? I've never seen these help message printed. I tried by 1) run move_group 2) run commander 3) stop move_group 4) commander.execute (passing a decent plan) but couldn't see this called (somehow commander just returned FAlSE).

**Comment by 130s** [comment]
(For my question above, maybe adding an answer as an inline comment (somewhere around [here](https://github.com/ros-planning/moveit_commander/pull/42/files#diff-3fad63fac0e8c61de84350f6122f8bceR539)?) might be the most effective for the future purpose?)

**Comment by gavanderhoorn** [comment]
@130s wrote:

> Sorry for ignorance but can you teach me how to see this portion of the code gets called? I've never seen these help message printed. I tried by 1) run move_group 2) run commander 3) stop move_group 4) commander.execute (passing a decent plan) but couldn't see this called (somehow commander just returned FAlSE).

iirc, you should just be able to start the commander and type `help` at the prompt.

**Comment by v4hn** [comment]
This works fine and looks nicer than before. +1 to merge into indigo, jade and kinetic branches
go ahead @davetcoleman 

Why the hack do we have all three branches if nothing changed... indigo-devel would have been perfectly enough (for now).

**Comment by davetcoleman** [comment]
Aside: There is no kinetic branch, actually. However I am a fan of making branches in each MoveIt repo for every ROS distro version in one big swoop so that new PRs can easily target the latest branch without having to request a maintainer to create the needed branch and then re-creating the PR.

**Comment by davetcoleman** [comment]
Cherry-picked to jade-devel: https://github.com/ros-planning/moveit_commander/commit/86a74ff73192c21ad680b9b6c6ffcbf970a78a5f

**Pull request merged**